### PR TITLE
[feature/GOVCMS-3905] Use TugboatQA to build PR test environments for distro/modules/themes

### PR DESCRIPTION
**Pull request by mingsong-hu**

### CODE REVIEW

**Comment by simesy** [comment]
AMDS can you please remove all the boilerplate comments from settings.php they just make it hard to eyeball a file and determine what is custom. Please see https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php

I usually link to the canonical settings file, which means we don't need maintain duplicate code. https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php#L8

**Comment by mingsong-hu** [comment]
> AMDS can you please remove all the boilerplate comments from settings.php they just make it hard to eyeball a file and determine what is custom. Please see https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php
> 
> I usually link to the canonical settings file, which means we don't need maintain duplicate code. https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php#L8

Sure. Just clean up all comments.

File: .tugboat/settings.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+ini_set('session.gc_probability', 1);
+ini_set('session.gc_divisor', 100);
+
+ini_set('session.gc_maxlifetime', 200000);
+
+ini_set('session.cookie_lifetime', 2000000);
+
+$conf['404_fast_paths_exclude'] = '/\/(?:styles)|(?:system\/files)\//';
+$conf['404_fast_paths'] = '/\.(?:txt|png|gif|jpe?g|css|js|ico|swf|flv|cgi|bat|pl|dll|exe|asp)$/i';
```
Comment (fubarhouse) [inline .tugboat/settings.php:25]: Not a show-stopper here, but why does this configuration deviate from [the original](https://github.com/govCMS/govcmslagoon/blob/develop/.docker/images/govcms7/settings/settings.php)?

Comment (mingsong-hu) [reply .tugboat/settings.php:25]: The "original" one is the settings for the Fast 404 module (https://www.drupal.org/project/fast_404) in govcmslagoon.

The govcms distribution does't include the Fast 404 module. 

The setting here is for the D7 core. More detail see this core API:
https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/drupal_fast_404/7.x

**Review by fubarhouse — APPROVED** [review]

**Comment by mingsong-hu** [comment]
> AMDS can you please remove all the boilerplate comments from settings.php they just make it hard to eyeball a file and determine what is custom. Please see https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php
> 
> I usually link to the canonical settings file, which means we don't need maintain duplicate code. https://github.com/govCMS/govCMS8/blob/1.x/.tugboat/settings.php#L8

Thanks @simesy , the reference to the canonical setting file is added to the beginning of the setting file.

**Pull request merged**